content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -41,32 +48,38 @@ $ helm upgrade istio install/kubernetes/helm/istio --namespace istio-system
 
 You can also use Kubernetes’ rolling update mechanism to upgrade the control plane components. This is suitable for cases when Istio hasn't been installed using Helm.
 
-First, generate the desired Istio control plane yaml file, e.g.
+1. Upgrade all the Istio's CRDs via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kube api-server:
 
 {{< text bash >}}
-$ helm template install/kubernetes/helm/istio --name istio \
-    --namespace istio-system > install/kubernetes/istio.yaml
+$ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
 {{< /text >}}
 
-or
+1. Render Istio's core components to a Kubernetes manifest file, e.g.
```

Comment:
It's unclear to the reader to identify what you mean with this step. I advise you change the verb used since "Render" might be too abstract. What exactly does the command do? Avoid using Latin abbreviations since they are difficult to understand too.
```suggestion
1. Add Istio's core components to a Kubernetes manifest file, for example.
```

Reply:
done